src/shared/vscode/env.ts

```diff
@@ -97,17 +97,42 @@ type ServiceConfig = Partial<{ [K in keyof ConfigToEnvMap]: any }>
 const logConfigsOnce: { [key: string]: ReturnType<typeof onceChanged> } = {}
 
 /**
- * Accepts a service name and a {config key -> expected env var name} map.
+ * Generate a map of environment variable names to environment variables.
+ *
+ * It does this by converting camel case entries in envVarNames to uppercase joined by underscores
+ * If there is no camel case in an entry in envVarNames then it just converts the envVar to uppercase
+ *
+ * E.g. if service is codecatalyst:
+ *  gitHostname -> __CODECATALYST_GIT_HOSTNAME
+ *  region -> __CODECATALYST_REGION
+ */
+function getEnvVars(service: string, envVarNames: string[]) {
```

Comment:
we probably need test coverage for this. and the test cases should probably cover all of the env vars that we care about.